recipes/pypolyagamma-3/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+if [ $(uname) == Darwin ]
+then
+    export CC=gcc CXX=g++
+fi
+
+if [ $(uname) == Linux ]
+then
+    yum install gsl-devel
+fi
+
+
+python setup.py install
```

Comment:
Also, we need the arguments `--single-version-externally-managed --record=record.txt` as `setuptools` is in use.